clients/ts/signalr-protocol-msgpack/README.md

```diff
@@ -15,9 +15,10 @@ To use the client in a browser, copy `*.js` files from the `dist/browser` folder
 ### Example (Browser)
 
 ```JavaScript
-let connection = new signalR.HubConnection('/chat', {
-    protocol: new signalR.protocols.msgpack.MessagePackHubProtocol()
-});
+let connection = new signalR.HubConnectionBuilder()
+    .withUrl("/chat")
+    .withHubProtocol(new signalR.protocols.msgpack.MessagePackHubProtocol())
```

Comment:
Double check this

Reply:
No, I just wrote it off the top of my head and didn't check at all

Reply:
Ah, I see what you meant now. I had fixed this locally but not pushed it.